.gitignore

```diff
@@ -41,3 +41,636 @@ user.bazelrc
 
```

Comment:
Is the `.gitignore` change intended (per commit message, it seems like no).
For that matter, the additions are around OS and IDE ignores. Have you considered [Global.gitignore](https://github.com/github/gitignore/tree/main/Global) for those? There are sources about it in the readme in the linked page, but the gist of it is that `Global.gitignore` contains ignores related to a particular user's environment (OS, IDE) and that way you don't have to rely on each repo you work with to have them.

Reply:
Ah sorry, I'm using gitignore.io to generate my gitignore files and uses Clion, MSVC as IDE(s)...

_I will checkout Global.gitignore, thx_